backend/demo_youtube_download.py

```
@@ -0,0 +1,188 @@
+from icpy.agent.tools.youtube_download_tool import YouTubeDownloadTool, YT_DLP_AVAILABLE
+
+
+async def demo_download(url: str, quality: str = 'medium'):
+    """Demo the YouTube download tool"""
+    
+    if not YT_DLP_AVAILABLE:
+        print("❌ yt-dlp is not installed. Please run: uv add yt-dlp")
+        return
+    
+    print(f"\n{'='*70}")
+    print(f"  YouTube Download Tool - Demo")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)